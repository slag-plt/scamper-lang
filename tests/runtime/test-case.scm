(test-case "simple equality success"
  equal? 1 1)

(test-case "simple equality failure"
  equal? 1 2)

(test-case "simple equality failure with computation"
  = (/ 10 2) (+ 1 2))

(test-case "epsilon equality success"
  (=-eps 0.001) 3 3.00001)

(test-case "epsilon equality failure"
  (=-eps 0.001) 3 5)