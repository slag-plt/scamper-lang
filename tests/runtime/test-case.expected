[[ Test "simple equality success" passed! ]]
[[ Test "simple equality failure" failed!
  Expected: 1
  Actual: 2
]]
[[ Test "simple equality failure with computation" failed!
  Expected: 5
  Actual: 3
]]
[[ Test "epsilon equality success" passed! ]]
[[ Test "epsilon equality failure" failed!
  Expected: 3
  Actual: 5
]]
